app/controllers/calendar_controller.rb

```diff
@@ -1,11 +1,21 @@
 class CalendarController < ApplicationController
 
   def index
-    @days = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri']
+    week_start = params[:start] ? Date.parse(params[:start]) : Date.today.beginning_of_week
```

Comment:
I would suggest passing only `week_start` in params and calculate week_end. Unless @syktus wants to display several weeks at once?